### PR TITLE
Update wizard doc steps

### PR DESCRIPTION
**Pull request by coulof**
Wrong info removed per request https://github.com/dell/csm-docs/pull/787/files#r1331661379

# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# Checklist:

- [ ] Have you run a grammar and spell checks against your submission?
- [ ] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 7. If your Kubernetes distribution doesn't have Volume Snapshot feature enabled, Refer to [this section](../../snapshots) to install the Volume Snapshot CRDs and the default snapshot controller.
 
-8. If Replication is checked in the wizard, only the sidecar is enabled. Refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters and the replication controller.
+8. If Replication is checked in the wizard, refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters.
```
Comment (santhoshatdell) [inline content/docs/deployment/csminstallationwizard/_index.md:86]: 'only sidecar is enabled' seems to be applicable only for Authorization module. Please see earlier version of doc - https://dell.github.io/csm-docs/v1/deployment/csminstallationwizard/
If this is the case, could you revert the change for other modules?

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 4. If Authorization is checked in the wizard, only the sidecar is enabled. Refer to [Authorization](../../authorization/deployment/helm/) to install and configure the CSM Authorization Proxy Server.
 
-5. If Replication is checked in the wizard, only the sidecar is enabled. Refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters and the replication controller.
+5. If Replication is checked in the wizard, refer to [Replication](../../replication/deployment/) on how to configure communication between Kubernetes clusters.
```
Comment (santhoshatdell) [inline content/docs/deployment/csminstallationwizard/_index.md:115]: Refer link should be reverted to previous one.
refer to [Replication](../csmoperator/modules/replication) for the necessary prerequisites required for this module.

Comment (coulof) [reply content/docs/deployment/csminstallationwizard/_index.md:115]: Sorry, I don't get it. Would you mind modifying the file directly to what you expect ?

Comment (santhoshatdell) [reply content/docs/deployment/csminstallationwizard/_index.md]: Updated.

File: content/docs/deployment/csminstallationwizard/_index.md [hunk 3]
```diff
@@ -108,13 +106,11 @@ The [Dell Container Storage Modules Installation Wizard](./src/index.html) is a
 
 >NOTE: The CSM Installation Wizard generates `values.yaml` with the minimal inputs required to install the CSM. To configure additional parameters in values.yaml, you can follow the steps outlined in [PowerStore](../../csidriver/installation/helm/powerstore/#install-the-driver), [PowerMax](../../csidriver/installation/helm/powermax/#install-the-driver), [PowerScale](../../csidriver/installation/helm/isilon/#install-the-driver), [PowerFlex](../../csidriver/installation/helm/powerflex/#install-the-driver), [Unity XT](../../csidriver/installation/helm/unity/#install-csi-driver), [Observability](../../observability/), [Replication](../../replication/), [Resiliency](../../resiliency/).
```
Comment (santhoshatdell) [inline content/docs/deployment/csminstallationwizard/_index.md:107]: @coulof Previous Note for Operator method had links to Operator doc section of drivers and modules (old line # 117). Newer note refers to the ones same as Helm method. Should we revert these links?

Comment (santhoshatdell) [reply content/docs/deployment/csminstallationwizard/_index.md]: Reverted the links.

**Review by santhoshatdell — APPROVED** [review]

**Review by gallacher — APPROVED** [review]
